.travis.yml

```diff
@@ -1,7 +1,14 @@
 sudo: required
 dist: trusty
 language: go
+env:
+  global:
+    - secure: "PkmY54fBn83ZHkDuortqgNOwR2JDlIMSEjD9yIYVEg1iMsHicA3BknM35kGbFCqlSn4GU8DTgJEKFXAZkVlanei+da2R24GH/kKbZQOUvclzTxJteev0rTm8ZJltXKTRnj/e/1UyeM2kum7fVrGwWpG7rQBi7SPF/ROM5jM+CMbI9R2g+LXruKNYmofSa9cZk8gYOXwaaXmXCm0d4UnJDesWcQ+4v+GuovnC8lkIKK9BjxbmWPVp8+cCtRkRxBkML9pZK9WUCXvgxlGf80F6N8vmgCzo9/LITJcSKmyO/OCQmhn9NnKHUCn8xLSjrZkWCL0JKZAzMhGXrmGhC17NnV3VZ8tfYVWrBZV5aVh2KK5vdYlW5sas0BgxBppTmFeYCetjs/VpAuCFlchV98sSgpnGSc9F5qEiez5DUWvvlVzjO9soHKtQeecgHrfk7Bgp9P0n1jvVEh3uFMK52tOPOF6mc9EDnivKa698381iD+ob3KC+IXYNmQlG36V4b0uhCrupp1PES/l6ZVjDrPVmd/J7UQJDLWdH2KmKkXwQIobBCyor4mlyA/j+EMm08cQ7uX5GfD+3zFrMqvOs7QIt+8RpGrSZGx4H9t7/w7BBAtYlOkrW2voPrvB0DgglvixWrVQ7hgj5cYGTLsxaemGcotiKEoGBVlV0pRRelesFxLE="
```

Comment:
It would be nice if it could be documented somewhere what process was followed to obtain the token and a link to the instructions how its generated (documented somewhere :)